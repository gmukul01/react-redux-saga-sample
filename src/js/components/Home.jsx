import React, { PureComponent, Fragment } from "react";
import NavigationBar from "components/NavigationBar";

export default class Home extends PureComponent {
  render() {
    return (
      <Fragment>
        <NavigationBar />
        <iframe className="iframe" src="/static/home.html" />
      </Fragment>
    );
  }
}
