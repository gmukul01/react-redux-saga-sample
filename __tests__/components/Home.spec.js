import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import Home from "components/Home";

describe("Home page ", () => {
  it("should render correctly", () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Home />);
    expect(result).toMatchSnapshot();
  });
});
