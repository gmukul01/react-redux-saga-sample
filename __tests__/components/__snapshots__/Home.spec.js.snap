// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page  should render correctly 1`] = `
<React.Fragment>
  <NavigationBar />
  <iframe
    className="iframe"
    src="/static/home.html"
  />
</React.Fragment>
`;
